lib/ferrum/frame/dom.rb

```diff
@@ -95,6 +95,22 @@ def body
         evaluate("document.documentElement?.outerHTML") || ""
       end
 
+      #
+      # Returns the element in which the window is embedded.
+      #
+      # @return [Node, nil]
+      #   The node in which the window is embedded.
+      #
+      # @example
+      #   browser.go_to("https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe")
+      #   frame = browser.frames.last
+      #   frame.node # => [Node]
+      #   frame.parent.parent.parent.node # => nil
+      #
+      def node
```

Comment:
I think the naming `node` is very confusing. What if we call this method on a node inside iframe? What node can be returned for a node? It should resemble js naming `frame_element` or `frame_node` or even both I think

Reply:
@route thanks for the feedback.

My original need for this PR was to try to get the `Ferrum::Node` for a given `Ferrum::Frame` - hence I chose to call the method `#node` to match the corresponding return type (I also considered `#to_node`, `#as_node`, `#to_element`).

However I can agree that the naming could lead to confusion. Mirroring Web APIs in Ferrum isn't a bad idea either - and will likely aid Ferrum's API discoverability.

I've gone ahead and renamed the new method `#frame_element`. I haven't added an alias for `#frame_node` as I'm not 100% sure offering a choice of methods makes for a better overall API - let me know if want me to make that change though.

Reply:
sounds good!